javascript/javascript3/week3/shoppingCart/shoppingCart.js

```diff
@@ -0,0 +1,166 @@
+class Product {
+    constructor(name, price) {
+        this.name = name;
+        this.price = price;
+    }
+
+    // added async method to convert the currency rate
+    async convertToCurrency(currency) {
+        try {
+            const fetchApi = await fetch(
+                'http://api.exchangeratesapi.io/v1/latest?access_key=59cc9db7a6c3570a340fc7bb48635976',
+            );
+            const apiResponse = await fetchApi.json();
+            const data = apiResponse.rates;
+
+            const currencyInArray = Object.entries(data); //converts objects into array so we can get every items key value pair in array.The Object.entries() method returns an array of arrays. fx. console.log(Object.entries(obj));
```

Comment:
if you only want an object keys name, you can use Object.keys() 
[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)

Reply:
Hi Ida, thanks for the feedback, I need both key and value here. When a person search for "USD" then I want key:USD and value:1.1 so that I can multiply with the provided price.